config/plugins.yaml

```diff
@@ -91,6 +91,10 @@ require_matching_label:
   prs: true
   regexp: ^kind/
 
+dco:
+  jetstack:
+    trusted_apps: [ "jetstack-bot" ]
```

Comment:
The documentation talks about not including the `bot` suffix....although I don't know how it would assume that all Bot accounts have that suffix:

https://github.com/kubernetes/test-infra/blob/a98f7f6910df237202851f6ab0b8b43dec2859b5/prow/plugins/config.go#L682-L690

> TrustedApps defines list of apps which commits will not be checked for DCO singoff.
> The list should contain usernames of each GitHub App without [bot] suffix.
> By default, this option is ignored.
>

Reply:
@wallrj could you approve the PR? We can only be 100% certain by testing this configuration 😄 .